ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -44,3 +44,43 @@ nginx_sites:
       balancer: "{{ deploy_env }}_formplayer/"
       proxy_next_upstream_tries: 1
       proxy_read_timeout: 900s
+    - name: /files
```

Comment:
I think it's worth it to not contaminate the other envs with all these locations.